primary/Tenkuu Danzai Skelter Heaven (primary).ass

```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
-Dialogue: 0,0:13:49.37,0:13:50.31,Default,,0,0,0,,But...
-Dialogue: 0,0:13:51.44,0:13:52.71,Default,,0,0,0,,those girls
-Dialogue: 0,0:13:53.21,0:13:54.40,Default,,0,0,0,,were smiling.
-Dialogue: 0,0:13:56.11,0:13:58.30,Default,,0,0,0,,That's why... That's why...
+Dialogue: 0,0:13:35.17,0:13:38.77,Default,,0,0,0,,We know its weak spot now, \Nso all I've gotta do is ram into it with my unit!{弱点さえ分かれば、こいつで突っ込むのもわけないさ}{pretty sure he means as a suicide attack, with the alternative in his mind being one of the girls doing the same thing}
+Dialogue: 0,0:13:38.77,0:13:40.73,Default,,0,0,0,,Funagai! Have you lost your mind?
+Dialogue: 0,0:13:42.17,0:13:43.90,Default,,0,0,0,,I told myself I was just doing my job.
+Dialogue: 0,0:13:45.61,0:13:48.71,Default,,0,0,0,,I coped with everything by convincing \Nmyself that was true.
+Dialogue: 0,0:13:49.37,0:13:52.71,Default,,0,0,0,,But thinking back on it...
+Dialogue: 0,0:13:53.21,0:13:54.40,Default,,0,0,0,,all I can see is their smiles.
+Dialogue: 0,0:13:56.11,0:13:58.30,Default,,0,0,0,,So, I... I—{Na: For some reason I get the feeling he's trying to say that he loves these robochicks, so making this a little more vague.}
 Dialogue: 0,0:14:13.50,0:14:15.93,Default,,0,0,0,,Rin! What are you doing?
-Dialogue: 0,0:14:15.93,0:14:20.34,Default,,0,0,0,,I didn't follow your orders all this time because of how I was created!
-Dialogue: 0,0:14:20.72,0:14:23.94,Default,,0,0,0,,It was because I loved you!
+Dialogue: 0,0:14:15.93,0:14:20.34,Default,,0,0,0,,I haven't followed your orders just \Nbecause I was created to do so!
+Dialogue: 0,0:14:20.72,0:14:23.94,Default,,0,0,0,,I've done it because I love you!
 Dialogue: 0,0:14:26.71,0:14:27.95,Default,,0,0,0,,That's right.
 Dialogue: 0,0:14:28.91,0:14:30.92,Default,,0,0,0,,I'm going in first!
 Dialogue: 0,0:14:36.15,0:14:37.18,Default,,0,0,0,,Don't!
 Dialogue: 0,0:14:37.86,0:14:39.65,Default,,0,0,0,,Stop it!
 Dialogue: 0,0:14:58.01,0:14:59.23,Default,,0,0,0,,I got through!
 Dialogue: 0,0:15:01.13,0:15:03.18,Default,,0,0,0,,Here we go, Rin!
-Dialogue: 0,0:15:06.22,0:15:09.96,Default,,0,0,0,,Skelter Titan and Skelter Heaven have penetrated the target.
-Dialogue: 0,0:15:11.72,0:15:12.86,Default,,0,0,0,,What on earth...
-Dialogue: 0,0:15:13.58,0:15:14.34,Default,,0,0,0,,What is this?
-Dialogue: 0,0:15:21.98,0:15:24.14,Default,,0,0,0,,This is the controlling core.
-Dialogue: 0,0:15:24.95,0:15:28.03,Default,,0,0,0,,It's speaking directly into my mind
-Dialogue: 0,0:15:31.44,0:15:35.38,Default,,0,0,0,,Certainly, some humans are slaves to their emotions.
-Dialogue: 0,0:15:35.38,0:15:39.04,Default,,0,0,0,,And their history is one of destruction, no end in sight.
+Dialogue: 0,0:15:06.22,0:15:09.96,Default,,0,0,0,,Skelter Titan and Skelter Heaven \Nhave penetrated the target.
+Dialogue: 0,0:15:11.72,0:15:14.34,Default,joined,0,0,0,,{\i1}What on earth... {Line 2: What is this?}
```

Comment:
Joining is fine, but we need to preserve the pause somehow.

Reply:
The audio does a fine job of preserving the pause. The subtitles don't have to reflect that. If these were closed captions, that'd be one thing, but they're not.